src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,9 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
```

Comment:
Same with map procces try to find a solution of how to use a `<ul>` with `<li>` here